test/test_code_action_corner_cases.vader

```diff
@@ -92,7 +99,7 @@ Execute(Preserve (no)eol at eof):
 
   " there doesn't seem to be a way to tell if a buffer is empty or contains one
   " empty line :-(
-  AssertEqual "", TestChanges("", [], 'file')
+  AssertEqual "\n", TestChanges("", [], 'file')
```

Comment:
This makes the above comment meaningless.

The point of that comment was that this test can't be done for the `save` and `buffer` modes, as these go through a buffer and change the file contents even though they should not. Now that all edits go through a buffer, not even the `file` mode works properly. At this point we might as well test all three modes and add a comment explaining that the behaviour is wrong and the only point of the test will be to catch changes in behaviour, not to test that it behaves correctly.

Reply:
Also, since all modes go through a buffer, it may not be necessary to test everything three times. The only differences now are whether the buffer is switched and whether it's saved (the test plumbing would save it anyway if it was meant to not be saved), none of which is relevant to these corner case tests any more.

Reply:
Thanks, I was going to ask you if it's okay to remove this line!

Reply:
Done!